mvc/_atBindingMixin.js

```diff
@@ -7,15 +7,13 @@ define([
 	"./resolve",
 	"./sync"
 ], function(array, lang, declare, has, Stateful, resolve, sync){
-	if(has("mvc-bindings-log-api")){
-		function getLogContent(/*dojo/Stateful*/ target, /*String*/ targetProp){
+	var getLogContent = (has("mvc-bindings-log-api")) ? function getLogContent(/*dojo/Stateful*/ target, /*String*/ targetProp){
```

Comment:
@dylans, not that it has a difference, but did you make this function expression a named one on purpose?